public/app/features/dashboard-scene/settings/EditListViewSceneUrlSync.ts

```diff
@@ -1,9 +1,12 @@
 import { SceneObjectUrlSyncHandler, SceneObjectUrlValues } from '@grafana/scenes';
 
 import { DashboardLinksEditView, DashboardLinksEditViewState } from './DashboardLinksEditView';
+import { VariablesEditView, VariablesEditViewState } from './VariablesEditView';
 
+type EditListViewUrlSync = DashboardLinksEditView | VariablesEditView;
+type EditListViewState = DashboardLinksEditViewState | VariablesEditViewState;
 export class EditListViewSceneUrlSync implements SceneObjectUrlSyncHandler {
-  constructor(private _scene: DashboardLinksEditView) {}
+  constructor(private _scene: EditListViewUrlSync) {}
```

Comment:
I think this needs to be something like `private _scene: SceneObject<DashboardEditListViewState>` to avoid the union types definition like above (i.e. `type EditListViewUrlSync = DashboardLinksEditView | VariablesEditView`). Overall the `EditListViewSceneUrlSync` should not care about particular type of the view, but just the fact that it's an indexable list.